clients/web/src/stylesheets/widgets/metadataWidget.styl

```diff
@@ -27,6 +27,46 @@
 .g-widget-metadata-value
   width 60%
 
+.g-widget-metadata-value pre
+  padding 0px
+  font-size 14px
+  color #777
+  background-color transparent
+  border none
+  max-height 320px
+  overflow auto
+
+.json-editor
+  width 55%
+  height 400px
+  display inline-block
+  background-color white
+  vertical-align middle
+  margin-left 5px
+  margin-right 5px
+
+.jsoneditor
+  border 2px solid #e3e3e3
+
+.jsoneditor .menu
+  background-color #fbfbfb
+  border-bottom 1px solid #f0f0f0
+
+.jsoneditor .menu button
+  background #fbfbfb url("../../../lib/jsoneditor/img/jsoneditor-icons.png");
+  border 1px solid #f0f0f0
+
+.jsoneditor .menu button:disabled
+  background #fbfbfb url("../../../lib/jsoneditor/img/jsoneditor-icons.png");
+  border 1px solid #f0f0f0
```

Comment:
The icons do not appear for me. This may be because I am mounting Girder not on the root level. I am able to see the icons when I change this and line 56 to

```
background #fbfbfb url("../lib/jsoneditor/img/jsoneditor-icons.png");
```